modules/tinymce/src/core/test/ts/browser/FormatterRemoveTest.ts

```diff
@@ -3,6 +3,7 @@ import { LegacyUnit, TinyAssertions, TinyHooks, TinySelections } from '@ephox/wr
 import { assert } from 'chai';
 
 import Editor from 'tinymce/core/api/Editor';
+import { ZWSP } from 'tinymce/src/core/main/ts/text/Zwsp';
```

Comment:
```suggestion
import { ZWSP } from 'tinymce/core/text/Zwsp';
```
Just noticed this import path isn't quite correct either sorry... You could also just use `Unicode.zwsp` directly if you preferred 🤷